### PR TITLE
Setup component calculator

### PR DESCRIPTION
**Pull request by touredri**
In this PR we have the following:
-Add new component 
-Make interface with JSX/CSS

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            value="="
+          >
+            =
+          </button>
+        </td>
+      </tr>
+    </table>
+  </div>
+);
+
```
Comment (VICTOR-CHUKWU) [inline src/components/calculator.js]: - [ ] kindly put this in a separate component and import it here, this is good for reusability and makes your code base more professional

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @touredri  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

- [x] kindly remove all boiler plate created by React like  `App.test.js`, `reportWebVitals.js`, `setupTests.js`, `robots.txt` and `manifest.json`
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 1 above)
Comment (touredri) [reply src/components/calculator.js]: Yeah good idea @VICTOR-CHUKWU

**Review by levy002 — APPROVED** [review]
## Hello @touredri 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- No linter errors :heavy_check_mark: 
- Implemented suggested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**